src/tests/test_magic_cmd.py

```diff
@@ -168,12 +168,69 @@ def test_table_profile(ip, tmp_empty):
 
     expected = {
         "count": [8, 8, 8, 8],
-        "mean": [12.2165, "6.875e-01", 88.75, 0.0],
-        "min": [10.532, 0.1, 82, ""],
-        "max": [14.44, 2.48, 98, "c"],
+        "mean": ["12.2165", "0.6875", "88.7500", math.nan],
+        "min": [10.532, 0.1, 82, math.nan],
+        "max": [14.44, 2.48, 98, math.nan],
+        "unique": [8, 7, 8, 5],
+        "freq": [math.nan, math.nan, math.nan, 4],
+        "top": [math.nan, math.nan, math.nan, "a"],
+    }
+
+    out = ip.run_cell("%sqlcmd profile -t numbers").result
+
+    stats_table = out._table
+
+    assert len(stats_table.rows) == len(expected)
+
+    for row in stats_table:
+        criteria = row.get_string(fields=[" "], border=False).strip()
+
+        rating = row.get_string(fields=["rating"], border=False, header=False).strip()
```

Comment:
There can be a common function `_get_row_string(column_name)` and use this to retrieve each row

Reply:
made this change added a helper function `_get_row_string(column_name)` that is used in three different pytest functions.